src/thermostat.js

```diff
@@ -173,7 +173,6 @@ export default function createThermostat({ Service, Characteristic }) {
         fn(...args)
         this.startDisconnectTimeout()
       }, (err) => {
-        err.isError = true
```

Comment:
the error was accessing property `isError` of string (since `err` which is returned from promise is string saying smth like "timeout blah-blah-blah".
so, that line was for sure errornous.
afaik, the homebridge puts the exclamation mark which shows that accessory is not operational in case there is something returned with the callback as first argument.
which I forgot to commit (but have it in my local copy) - will add it.